### PR TITLE
[Test] Add tenant cmd test

### PR DESCRIPTION
**Pull request by nodece**
Signed-off-by: Zixuan Liu <nodeces@gmail.com>

### Motivation

Add tests for the pulsar-admin tool.

### Modifications

- Add tenant cmd test in the integration module

### Documentation
 
- [x] `no-need-doc`

Add test

### CODE REVIEW

**Review by codelipenghui — DISMISSED** [review]
Great work!

File: tests/integration/src/test/java/org/apache/pulsar/tests/integration/cli/tenant/TenantTest.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.pulsar.common.policies.data.TenantInfo;
+import org.apache.pulsar.tests.integration.docker.ContainerExecResult;
+import org.apache.pulsar.tests.integration.suites.PulsarTestSuite;
+import org.testng.annotations.Test;
+
+public class TenantTest extends PulsarTestSuite {
```
Comment (codelipenghui) [inline tests/integration/src/test/java/org/apache/pulsar/tests/integration/cli/tenant/TenantTest.java:35]: Need to add the new test to `pulsar-cli.xml`

**Review by mattisonchao — DISMISSED** [review]

File: tests/integration/src/test/java/org/apache/pulsar/tests/integration/cli/tenant/TenantTest.java (hunk 1 above)
Comment (nodece) [reply tests/integration/src/test/java/org/apache/pulsar/tests/integration/cli/tenant/TenantTest.java:35]: @codelipenghui Do you mean that add a module named `pulsar-cli`?

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**